lib/parser.js

```diff
@@ -98,9 +100,18 @@ function _mapSymbol(symbol){
     symbol.type = types[0].types.join(' | ');
   }
 
-  if(returns.length === 1){
-    symbol["return"] = returns[0].types.join(' ');
-    symbol["returnDescription"] = returns[0].description.replace(/(<([^>]+)>)/ig, '');
```

Comment:
Hey @FGRibreau *returnDescription* never made it into the template. I only edited parser.js

this is my @return section in my local version of the jade template:

## @return
```
if symbol.returns
  table.table.table-bordered.table-striped
    thead
      tr
        th(style="width:20%") @return
        th Description
    tbody
      each tag in symbol.tags
        if tag.type == 'return'
          tr
            td= tag.types.toString()
            td= tag.description.replace(/(<([^>]+)>)/ig, '')
```